plugins/CorePluginsAdmin/vue/src/FormField/FieldExpandableSelect.vue

```diff
@@ -48,7 +48,7 @@
           <ul v-show="showCategory === options.group || searchTerm" class="collection secondLevel">
             <li
               class="expandableListItem collection-item valign-wrapper"
-              v-for="children in options.values.filter(x => x.value.indexOf(searchTerm) !== -1)"
+              v-for="children in options.values.filter(x => searchRegExp.test(x.value))"
```

Comment:
This would actually enable the possibility to use regular expressions in the search box. The problem is that it would throw an error if you type an invalid regex like `[a` in the search box.
To only change that to a insensitive match I guess the simples solution would be to apply a `toLowerCase` on both strings.

Reply:
@sgiehl that make sense, should reverse to that https://github.com/matomo-org/matomo/pull/18643/commits/70dd0ed253e5e92e9cb6c23a760aa515ccb9b307 solution. Was trying to make the function short :)